src/core/model.js

```diff
@@ -239,15 +223,15 @@ export default class Model extends Object3D {
     return Object.assign({}, uniforms, cameraUniforms, viewUniforms);
   }
 
-  /*
-   * @param {Camera} opt.camera=
-   * @param {Camera} opt.viewMatrix=
-   */
-  /* eslint-disable max-statements */
-  render(uniforms = {}) {
+  draw({uniforms = {}, attributes = {}, settings = {}, samplers = {}}) {
```

Comment:
Is the draw() function here for compatibility reasons? I see that it just calls Model.render() with all arguments passed over. 

Also, it seems that all deck.gl layers calls, Model.render().